calendar_public_holiday/models/calendar_public_holiday.py

```diff
@@ -0,0 +1,126 @@
+# Copyright 2015 2011,2013 Michael Telahun Makonnen <mmakonnen@gmail.com>
+# Copyright 2020 InitOS Gmbh
+# Copyright 2024 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import datetime
+
+from odoo import api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class ResourceCalendarPublicHoliday(models.Model):
+    _name = "calendar.public.holiday"
+    _description = "Calendar Public Holiday"
+    _rec_name = "year"
```

Comment:
We should put first the most modern one.

```suggestion
    _rec_name = "year desc"
```